content-repo/extra-docs/articles/microsoft-integrations-overview.md

```diff
@@ -0,0 +1,187 @@
+
+Review this document to determine the Microsoft integrations you need for your use case.
+
+
+## Azure Active Directory
+
+### Use Cases
+
+* Create users and groups.
+* Remove a member from a group.
+* Remove a pre-approved application.
+
+
+### [Azure Active Directory Users](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-user)
+Manage users in Azure Active Directory and O365.
+
+- List, create, and update users.
+- Terminate sessions.
+- Block users, etc.
+
+### [Azure Active Directory Groups](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-groups)
+Manage groups in Azure Active Directory and O365.
+- List, create, and update groups.
+- List, add, and remove members.
+
+### [Azure Active Directory Identity And Access](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-identityand-access)
+Manage Active Directory roles and role members.
+
+### [Azure Active Directory Applications / Service Principals](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-applications)
+Manage applications and service principals.
+
+## O365
+
+### Use Cases
+
+* Download a file from OneDrive.
+* Send a message via Microsoft Teams.
+* Add a member to an existing team.
+* Schedule an event in the calendar.
+
+### [O365 File Management (Onedrive/Sharepoint/Teams)](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-files)
+Manage files in O365 (OneDrive/SharePoint/Teams).
+- Upload and download files.
+- List drive and folder content.
+- List SharePoint sites.  
+
+### [Microsoft Teams](https://xsoar.pan.dev/docs/reference/integrations/microsoft-teams)
+Enable communicating and mirroring via Microsoft Teams.
+- Create and update channels.
+- Add users to channel.
+- Message users.
+- Ring user.
+- Message mirroring.
+
+### [Microsoft Teams Management](https://xsoar.pan.dev/docs/reference/integrations/microsoft-teams-management)
+Manage teams and team members
+- Create and update teams.
+- Add and remove team members.
+
+### [O365 Outlook Calendar](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-calendar)
+Manage calendar events.
+
+### [Microsoft Graph API](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-api)
+This is a generic integration that supports running any endpoint of MS Graph API.
+Since the API is very vast and not all of the endpoints are implemented, this integration can be used.
+
+
+## Audit Logs
+
+### Use Cases:
+* Find failed login events
+* Find publicly shared files events
+* Find security events
+
+### [Microsoft Management Activity](https://xsoar.pan.dev/docs/reference/integrations/microsoft-management-activity-api-o365-azure-events)
+Ingest events from O365 (Azure AD, SharePoint, EWS, etc) as incidents.
+
+### [Microsoft Policy And Compliance - Office365 And Azure Audit Log](https://xsoar.pan.dev/docs/reference/integrations/microsoft-policy-and-compliance-audit-log)
```

Comment:
The pack MicrosoftPolicyAndCompliance contains 2 playbooks only (without any integration), im going to move those playbooks to Office365AndAzureAuditLog pack.
So, please remove the Microsoft Policy And Compliance  since it will be deprecated.

Reply:
@altmannyarden done